sdk/opa_test.go

```diff
@@ -664,3 +666,57 @@ func TestOpaRuntimeConfig(t *testing.T) {
 		t.Fatalf("expected %v but got %v", exp, result.Result)
 	}
 }
+
+func TestPrintStatements(t *testing.T) {
+
+	ctx := context.Background()
+
+	s := sdktest.MustNewServer(sdktest.MockBundle("/bundles/b.tar.gz", map[string]string{
+		"x.rego": `package foo
+
+		p { print("XXX") }`,
+	}))
+
+	defer s.Stop()
+
+	config := []byte(fmt.Sprintf(`{
```

Comment:
I've recently changed all these to plain strings, and strings.NewReader. that's why merging main into this branch broke stuff